alchemist-loading/src/test/kotlin/it/unibo/alchemist/test/TestExportProperty.kt

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) 2010-2024, Danilo Pianini and contributors
+ * listed, for each module, in the respective subproject's build.gradle.kts file.
+ *
+ * This file is part of Alchemist, and is distributed under the terms of the
+ * GNU General Public License, with a linking exception,
+ * as described in the file LICENSE in the Alchemist distribution's top directory.
+ */
+
+package it.unibo.alchemist.test
+
+import io.kotest.core.spec.style.StringSpec
+import io.kotest.matchers.shouldBe
+import it.unibo.alchemist.boundary.LoadAlchemist
+import it.unibo.alchemist.boundary.exporters.CSVExporter
+import it.unibo.alchemist.boundary.exporters.GlobalExporter
+import it.unibo.alchemist.core.Simulation
+import it.unibo.alchemist.model.Position
+import org.junit.jupiter.api.Assertions
+import org.kaikikm.threadresloader.ResourceLoader
+
+class TestExportProperty<T, P : Position<P>> : StringSpec({
+    "test export property" {
+        val file = ResourceLoader.getResource("testExportProperty.yml")
+        Assertions.assertNotNull(file)
+        val loader = LoadAlchemist.from(file)
+        Assertions.assertNotNull(loader)
+        val simulation: Simulation<T, P> = loader.getDefault()
+        val exporters = simulation.outputMonitors.filterIsInstance<GlobalExporter<T, P>>().flatMap { it.exporters }
+        exporters.size shouldBe 1
+        val exporter = exporters.first()
+        require(exporter is CSVExporter) {
+            "Invalid exporter of type '${exporter::class.simpleName}'"
+        }
+    }
+})
```

Comment:
There must be a folder where you put the yaml and it gets tested automatically (maybe `alchemist-loading/src/test/resources/regression/`?)

Reply:
I didn't find anything inside the folder `alchemist-loading`.
Should it look like `./src/test/kotlin/TestWebsiteCodeSnippets.kt`? Which test all the yaml files in a certain folder?

Reply:
I think you can use `TestGuidedTourLoading`, simply add the yaml inside `guidedTour` and see if it works

Reply:
Test added

Reply:
And does it work? Namely, if you don't patch, does it fail correctly?

Reply:
Yes. 
First, I run the test without changing anything in the code base and it failed (IndexOutOfBound - MoleculeReader.kt - Line 68).
Changing the range in the substring method, as you suggested, made the test pass.